pkg/console/service/service_test.go

```diff
@@ -24,6 +25,7 @@ import (
 	"kubevirt.io/client-go/kubecli"
 
 	api "github.com/kubevirt/vm-console-proxy/api/v1alpha1"
+	fake_auth "github.com/kubevirt/vm-console-proxy/pkg/console/auth-config/fake"
```

Comment:
Avoid snake case?

Reply:
The same as with package names. I prefer separating words.

Reply:
Changed.